operations/image_management/Configure_a_Remote_Build_Node.md

```diff
@@ -688,3 +688,106 @@ Expected output will be something like:
     }
 ]
 ```
+
+### Checking the status of remote build nodes
+
+Before a job is set up to be launched on a remote build node, IMS checks that there is
+a remote node capable of running the job. If a node is registered as a remote build node
+but is not able to run remote jobs, that node will not have remote jobs scheduled on it.
+The status of the remote build nodes may be queried. The status object will have a field
+`ableToRunJobs` that contains a boolean indicating if the node is able to run jobs. If the
+node is not able to run jobs, the status object will document the issue with running jobs
+on that node.
+
+In order to run jobs all of the following conditions must be met:
+
+* The IMS service can SSH to the node.
+* The node architecture must be determined and valid.
+* The `podman` executable must be installed.
+
+(`ncn-mw#`) To check the status of a particular remote build node:
+
+```bash
+cray ims remote-build-nodes status describe "${IMS_REMOTE_NODE_XNAME}"
+```
+
+If the node is ready to run jobs, the output will look something like:
+
+```text
+{
+  "ableToRunJobs": true,
+  "nodeArch": "x86_64",
+  "numCurrentJobs": 4,
+  "podmanStatus": "Podman present at /usr/bin/podman",
+  "sshStatus": "SSH connection established.",
+  "xname": "x3000c0s3b0n0"
+}
+```
+
+If IMS is unable to SSH to a node the output may look something like:
+
+```text
+{
+  "ableToRunJobs": false,
+  "nodeArch": "Unknown",
+  "numCurrentJobs": -1,
+  "podmanStatus": "Unknown",
+  "sshStatus": "Unable to connect to node. Error: [Errno -2] Name does not resolve",
+  "xname": "x3000c0s3b0n1"
+}
+```
+
+If IMS is unable to determine the architecture of the node the output may look something like:
+
+```text
+{
+  "ableToRunJobs": false,
+  "nodeArch": "Unable to determine architecture of node.",
+  "numCurrentJobs": -1,
+  "podmanStatus": "Unknown",
+  "sshStatus": "SSH connection established.",
+  "xname": "x3000c0s3b0n1"
+}
+```
+
+If `podman` is not correctly installed the output may look something like:
+
+```text
+{
+  "ableToRunJobs": false,
+  "nodeArch": "x86_64",
+  "numCurrentJobs": -1,
+  "podmanStatus": "Podman not installed on node.",
+  "sshStatus": "SSH connection established.",
+  "xname": "x3000c0s3b0n1"
+}
```

Comment:
Do you need a fourth example where the number of jobs could not be determined?

Reply:
No - it will still run on a node where the number of jobs can't be determined.